marathon_acme/clients.py

```diff
@@ -226,41 +226,35 @@ def raise_for_not_ok_status(response):
     return response
 
 
-def sse_content_with_protocol(response, callbacks):
+def sse_content_with_protocol(response, handler):
```

Comment:
I think the reason I didn't add a `_` was because it was imported in a test. But it probably makes more sense to have a `_`.